packages/backend/src/graphql/resolvers/walletAddressKey.ts

```diff
@@ -4,10 +4,46 @@ import {
   WalletAddressKey as SchemaWalletAddressKey,
   Alg,
   Kty,
-  Crv
+  Crv,
+  WalletAddressResolvers
 } from '../generated/graphql'
 import { ApolloContext } from '../../app'
 import { WalletAddressKey } from '../../open_payments/wallet_address/key/model'
+import { Pagination } from '../../shared/baseModel'
+import { getPageInfo } from '../../shared/pagination'
+
+export const getWalletAddressKeys: WalletAddressResolvers<ApolloContext>['walletAddressKeys'] =
+  async (
+    parent,
+    args,
+    ctx
+  ): Promise<ResolversTypes['WalletAddressKeyConnection']> => {
+    const walletAddressKeyService = await ctx.container.use(
+      'walletAddressKeyService'
+    )
+
+    const getPageFn = (pagination_: Pagination) => {
+      if (!parent.id) throw new Error('missing wallet address id')
```

Comment:
could we do this check as the first thing in the resolver? (then we could probably get rid of `getPageFn` altogether)